lib/v2/datasource/remote/api/balance_repository.dart

```diff
@@ -29,4 +29,12 @@ class BalanceRepository extends NetworkRepository {
             }))
         .catchError((dynamic error) => mapHttpError(error));
   }
+
+  Future<List<Result<dynamic>>> getTokenBalances(String userAccount, List<TokenModel> tokens) {
```

Comment:
This should not be part of the balance repository. This goes into a use case. 

`getTokenBalance` is a real balance repo method. 

if we had a network call that could return multiple tokens, then it would be here.

Reply:
Umm... I guess I don't really understand the benefit of that. But, Ok.

Reply:
Saw some other places do this, did the same. 

Done.